consensus/types/src/beacon_state.rs

```diff
@@ -1091,6 +1095,12 @@ impl<E: EthSpec> BeaconState<E> {
 
         let seed = self.get_seed(epoch, Domain::SyncCommittee, spec)?;
 
+        let max_effective_balance = if self.fork_name_unchecked() >= ForkName::Electra {
+            spec.max_effective_balance_electra
+        } else {
+            spec.max_effective_balance
```

Comment:
Should we make `max_effective_balance` a function to prevent these typos?

Reply:
i think that would only do us any good if we made `max_effective_balance` private so it's only accessible via the function. Would you wanna do that?

Reply:
How many places is `max_effective_balance` used in a context that does not require to be fork aware?

Reply:
Okay I've used a function for this that already existed on the `spec` object. I haven't yet made these fields private because the code in `validator.rs` is super convoluted and needs to be refactored to support that. Just don't have time to do it in this PR as we need to get `devnet-1` out and we only have 1 day and i gotta get working on the ef-tests